test/test_helper.rb

```diff
@@ -33,18 +33,12 @@ def stub_omniauth
     })
   end
 
-  def create_turing
-    turing = Place.create(
-      name: "Turing School",
-      place_id: "abc123",
-      image_url: "www.dono.com",
-      rating: "3",
-      phone_no: "(303) 421-2345",
-      website: "turing.io",
-      hours: "PEOPLE ARE ALWAYS HERE" 
-    )
-    Address.create(place_id: turing.id)
-    turing
+  def login_create_place
+    login_user
+      visit "/"
+      page.fill_in 'nav-search', 
+        :with => 'Turing School of Software & Design'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.